src/model/model.rs

```diff
@@ -355,7 +371,20 @@ impl Matcher for AddressPattern {
                 let bmask = !0u128 << (128 - prefix);
                 u128::from(*addrp) & bmask == u128::from(*addr) & bmask
             }
-            (P::Domain { pattern }, Address::Domain(domain, _)) => pattern.is_match(domain),
+            (P::Domain(DomainPattern::Regex { pattern }), Address::Domain(domain, _)) => {
+                pattern.is_match(domain)
+            }
+            (P::Domain(DomainPattern::Wildcard { wildcard }), Address::Domain(domain, _)) => {
+                let wildcard_after_escape = r"\*";
```

Comment:
https://github.com/Idein/gatekeeper/pull/44#discussion_r481849349
Can you change the name to something meaningful or to hard coded?

Reply:
done with 140ea88. (hard-coded)